tests/test_basic_example_v2.py

```diff
@@ -47,7 +47,7 @@ def testmodulev2_venv(pytestconfig) -> Iterator[env.VirtualEnv]:
 
 @pytest.fixture(scope="function")
 def testmodulev2_venv_active(
-    deactive_venv: None,
+    deactive_venv: Iterator[None],
```

Comment:
I don't think this is correct. The return type of the fixture is abc.Iterator[None], but the value of the argument here is None, isn't it?

Reply:
Right, indeed :+1: